libs/ui/src/button.tsx

```diff
@@ -28,15 +28,6 @@ export interface StyledButtonProps {
   variant?: ButtonVariant;
   icon?: IconName | JSX.Element;
   rightIcon?: IconName | JSX.Element;
-
-  /** @deprecated Place the button within a flex or grid container instead. */
-  readonly fullWidth?: boolean;
-
-  /** @deprecated Incompatible with new VVSG size themes */
-  readonly large?: boolean;
-
-  /** @deprecated Incompatible with new VVSG size themes */
-  readonly small?: boolean;
```

Comment:
These were truly not doing anything at all? 

😮 

Scandalous.

Reply:
I think they were doing things long ago, but were deprecated and then unimplemented at some point along the way